src/helpers/Webp.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "../defines.hpp"
+#include <webp/decode.h>
```

Comment:
no point in including webp here, can be done in the source file.

Reply:
Ok, I will move it to source file.